specification/trace/semantic_conventions/tls.md

```diff
@@ -0,0 +1,80 @@
+# Semantic conventions for TLS spans
+
+**Status**: [Experimental](../../document-status.md)
+
+This document defines semantic conventions for TLS/SSL client and server Spans.
```

Comment:
Can you add more descriptive bits here, e.g. WHEN these apply?

e.g. How does this work when a "front" server does SSL encryption, and all the various ways the handshake can work?

The HTTP semantic conventions provide examples which I think are useful for this task, e.g. https://github.com/open-telemetry/semantic-conventions/blob/main/specification/trace/semantic_conventions/http.md#examples

Reply:
@svrnm thanks for the explanation!

I have no objections to having TLS/SSL telemetry as spans if local timing is important. 

> See how it is implemented in JavaScript, the context can be propagated in code. The TLS span is just a wrapper around the HTTP server span.

My understanding is that the handshake happens before HTTP request starts (and there could be no HTTP request in the case of other protocols). so we probably won't be able to correlate HTTP and TLS spans/logs in a general case (languages, client libraries, etc) on the server side. 

as @jsuereth pointed out, it'd be great to see examples of client and server TLS span like these: https://github.com/open-telemetry/semantic-conventions/blob/main/specification/trace/semantic_conventions/http.md#examples with some examples of how they can be related to HTTP spans

Reply:
Thinking about this for a while. I am stuck with the following thought and I would like to get your input on that @lmolkova & @jsuereth:

if we look at existing spans representing networking requests (e.g. HTTP), we only have a single span for the top level communication (here HTTP), while there is no parent span representing the TCP connection & it's related timing (or the DNS request that may be happening within the HTTP call to resolve the remote host). TLS/SSL is very similar to that. So my question is now, if this is by design and there is a section in the spec calling this out or if this is not yet specified?

Reply:
I will resolve this conversation now, as it is not relevant for this PR anymore, please reopen it or object if this is not the case @lmolkova @jsuereth